.github/workflows/github_actions.yml

```diff
@@ -0,0 +1,102 @@
+name: TileDB-Trino
+
+on:
+  push:
+    tags:
+      - '*'
+    branches:
+      - 'master'
+  pull_request:
+
+jobs:
+  Test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-18.04, macos-10.15 ]
+
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          unset SYSTEM
+          set -e pipefail
+          open /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg
+          sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -allowUntrusted -target /
```

Comment:
Is this still necessary? I removed it in tiledb-py recently and everything still worked.

Reply:
It is not! I removed it.